src/SplitPackages/Program.cs

```diff
@@ -227,7 +227,7 @@ private ProjectJsonFileBuilder CreateBaseJsonFileBuilderForCache(string destinat
 
             var hardcodedDependencies = new[]
             {
-                new PackageInformation(null, "Microsoft.NetCore.App", "1.0.0", new[] { Frameworks.NetCoreApp10 })
+                new PackageInformation(null, "Microsoft.NetCore.App", "1.1.0", new[] { Frameworks.NetCoreApp10 })
```

Comment:
Would this affect tools? We're still trying to figure out if tools should continue targeting `Microsoft.NETCore.App 1.0.0`.